test/integration/stop/test_stop.py

```diff
@@ -0,0 +1,137 @@
+import os
+import shutil
+from sys import flags
+
+import pytest
+
+from utils import (pid_file, run_path, wait_file, run_command_and_get_output)
+
+
+def test_stop(tt_cmd, tmpdir_with_cfg):
+    shutil.copy(os.path.join(os.path.dirname(__file__), "test_app.lua"), tmpdir_with_cfg)
+    app_name = "test_app"
+    start_cmd = [tt_cmd, "start", app_name]
+    rc,out = run_command_and_get_output(start_cmd, cwd=tmpdir_with_cfg)
+    assert "Starting an instance" in out
+    assert wait_file(os.path.join(tmpdir_with_cfg, app_name, run_path, app_name),
+                     pid_file, []) != ""
+
+    try:
+        # Test confirmed stop.
+        stop_cmd = [tt_cmd,"stop",app_name]
+        rc,out = run_command_and_get_output(stop_cmd,cwd=tmpdir_with_cfg,input="y\n")
+        assert f"Confirm stop of '{app_name}' [y/n]" in out
+        assert "has been terminated" in out
+        assert wait_file(os.path.join(tmpdir_with_cfg, app_name, run_path, app_name),
+                         pid_file, []) == ""
+
+        start_cmd = [tt_cmd, "start", app_name]
+        rc,out = run_command_and_get_output(start_cmd, cwd=tmpdir_with_cfg)
+        assert rc == 0
+        assert "Starting an instance" in out
+        assert wait_file(os.path.join(tmpdir_with_cfg, app_name, run_path, app_name),
+                         pid_file, []) != ""
+
+        # Test cancelled stop.
+        stop_cmd = [tt_cmd,"stop",app_name]
+        rc,out = run_command_and_get_output(stop_cmd, cwd=tmpdir_with_cfg, input="n\n")
+        assert rc == 0
+        assert f"Confirm stop of '{app_name}' [y/n]" in out
+        assert "Stop is cancelled" in out
+        assert wait_file(os.path.join(tmpdir_with_cfg, app_name, run_path, app_name),
```

Comment:
This is not start, so we shouldn't wait for it, just check that it is not removed.